food-for-thought/app/map.tsx

```diff
@@ -67,9 +74,45 @@ const RestaurantMap = () => {
   const filterTypes = ['diets', 'allergens', 'ingredients', 'cuisine', 'meals'];
   const snapPoints = useMemo(() => ['25%', '50%'], []);
 
+  const pic = require('../assets/images/react-logo.png'); // placeholder restaurant image
+
+  const HOST_IP = Constants.expoConfig?.extra?.HOST_IP;
+
+  const loadUser = React.useCallback(async () => {
+    const token = await AsyncStorage.getItem('token');
+    if (token) {
+        try {
+            const decodedToken: any = jwtDecode(token);
+            console.log(decodedToken.username);
```

Comment:
not important, but would remove stray console logs (good to keep console clean ✨)

Reply:
Absolutely agree! We should remove these before merging - there's some other stray console logs on other pages as well, will look into tidying up those too🙏